ble_controller/doc/README.rst

```diff
@@ -0,0 +1,116 @@
+BLE Controller
+==============
+
+| The BLE Controller is an RTOS-agnostic library built for the Nordic
+  Semiconductor nRF52 Series that supports Bluetooth 5.
+| In addition to providing a BLE HCI interface, it also provides:
+
+-  Multiprotocol support through APIs for requesting RADIO timeslots
+-  APIs for accessing flash memory concurrently with radio activity
+-  APIs for controlling the high frequency clock
+
+| The BLE Controller is distributed in different variants containing
+  different features.
+| All variants are available as soft-float, softfp-float, and hard-float
+  builds.
+
+SoC support:
+
++------------+--------+--------+--------+
+|            | s112   | s132   | s140   |
++============+========+========+========+
+| nRF52810   | X      |        |        |
++------------+--------+--------+--------+
+| nRF52832   | X      | X      |        |
++------------+--------+--------+--------+
+| nRF52840   |        |        | X      |
++------------+--------+--------+--------+
+
+Feature support:
+
++--------------------------+--------+--------+--------+
+|                          | s112   | s132   | s140   |
++==========================+========+========+========+
+| 2 Mbps PHY               | X      | X      | X      |
++--------------------------+--------+--------+--------+
+| Advertiser               | X      | X      | X      |
++--------------------------+--------+--------+--------+
+| Slave                    | X      | X      | X      |
++--------------------------+--------+--------+--------+
+| Scanner                  |        | X      | X      |
++--------------------------+--------+--------+--------+
+| Master                   |        | X      | X      |
++--------------------------+--------+--------+--------+
+| Data Length Extensions   |        | X      | X      |
++--------------------------+--------+--------+--------+
+| Advertising Extensions   |        | X      | X      |
++--------------------------+--------+--------+--------+
+| Coded PHY (Long Range)   |        |        | X      |
++--------------------------+--------+--------+--------+
+
+*Disclaimer*
+
+-  The libraries are for evaluation purposes only.
+-  The libraries are not fully functional and not yet fully tested.
+-  The libraries are not optimized for performance.
+
+Integration guide
+-----------------
+
+Applications utilizing the BLE Controller library are responsible for
+the following:
+
+-  Forward RTC0, TIMER0, RADIO, POWER\_CLOCK, and RNG interrupts to the
+   BLE Controller library.
+   The application should not add additional processing in the
+   corresponding interrupt handlers,
+   otherwise the behavior is undefined.
+-  Call a signal handler when the low priority task processing interrupt occurs.
+-  Ensure thread safe operation. This is achieved by either:
+
+   -  Call all APIs from the same execution priority.
```

Comment:
Calling , Ensuring - at the beginning of the bullet points in this list

Reply:
This one I missed. Fixing it internally now.